src/lib/app-logs/app-logs.js

```diff
@@ -13,24 +13,34 @@ import gql from 'graphql-tag';
  */
 import API from 'lib/api';
 
+export const LIMIT_MAX = 5000;
+
 const QUERY_ENVIRONMENT_LOGS = gql`
-	query GetAppLogs( $appId: Int, $envId: Int, $type: AppEnvironmentLogType, $limit: Int ) {
+	query GetAppLogs( $appId: Int, $envId: Int, $type: AppEnvironmentLogType, $limit: Int, $after: String ) {
 		app( id: $appId ) {
 			environments( id: $envId ) {
 				id
-				logs( type: $type, limit: $limit ) {
+				logs( type: $type, limit: $limit, after: $after ) {
 					nodes {
 						timestamp
 						message
 					}
+					nextCursor
+					pollingDelaySeconds
 				}
 			}
 		}
 	}
 `;
 
-export async function getRecentLogs( appId: number, envId: number, type: string, limit: number ): Promise<Array<{ timestamp: string, message: string }>> {
-	const api = await API();
+type GetRecentLogsResponse = {
+	nodes: Array<{ timestamp: string, message: string }>,
+	nextCursor: string,
+	pollingDelaySeconds: number,
+};
+
+export async function getRecentLogs( appId: number, envId: number, type: string, limit: number, after?: string ): Promise<GetRecentLogsResponse> {
+	const api = await API( false );
```

Comment:
In case we change the param to an options object:
```suggestion
	const api = await API( { exitOnError: false } );
```